cirq/ops/gate_features.py

```diff
@@ -108,7 +104,4 @@ def num_qubits(self) -> int:
         return self._num_qubits
 
     def validate_args(self, qubits):
-        if len(qubits) != self.num_qubits():
-            raise ValueError(
-                '{}-qubit gate was applied to {} qubits'.
-                    format(self.num_qubits(), len(qubits)))
+        super().validate_args(qubits)
```

Comment:
Just delete the method if it forwards directly to `super()`.

Reply:
This entire class was deleted by an incoming merge from master.